gm4_display_frames/data/gm4_display_frames/functions/check_splash.mcfunction

```diff
@@ -0,0 +1,4 @@
+# @s = none
+# run from spawn_aec
+
+execute at @e[type=area_effect_cloud,tag=gm4_df_potion_tracker] unless entity @e[tag=gm4_df_potion_tracker,distance=.001..1] if entity @e[type=item_frame,tag=!gm4_df_invis_frame,distance=..2] run function gm4_display_frames:create
```

Comment:
```suggestion
execute at @e[type=area_effect_cloud,tag=gm4_df_potion_tracker] unless entity @e[type=area_effect_cloud,tag=gm4_df_potion_tracker,distance=.001..1] if entity @e[type=item_frame,tag=!gm4_df_invis_frame,distance=..2] run function gm4_display_frames:create
```
Adding an entity type here will drastically improve lookup time. Assuming all `gm4_df_potion_tracker` are `type=area_effect_cloud`.

Reply:
they’re not all area effect clouds, but we can add an entity type tag (splash/lingering potions along with AECs)

Reply:
That sounds like a good idea!